sdk/typescript/src/types/sui-bcs.ts

```diff
@@ -69,6 +69,17 @@ export type TransferSuiTx = {
   };
 };
 
+/**
+ * Transaction type used for Pay transaction.
+ */
+export type PayTx = {
+  Pay: {
+    input_coins: string[];
```

Comment:
can we rename this to `coins` so its consistent with sui-core?
https://github.com/MystenLabs/sui/blob/main/crates/sui-core/tests/staged/sui.yaml#L391-L403

Reply:
on RPC and CLI, it has been named as `input_coins` to be explicit, will stick with that.